package.json

```diff
@@ -1,77 +1,39 @@
-    "url": "https://github.com/imagekit-developer/imagekit-react"
-  },
-  "main": "dist/imagekitio-react.js",
-  "module": "dist/imagekitio-react.esm.js",
-  "browser": "dist/imagekitio-react.umd.js",
-  "browser:min": "dist/imagekitio-react.umd.min.js",
-  "files": [
-    "dist",
-    "src"
-  ],
-  "jest": {
-    "verbose": true,
-    "setupFilesAfterEnv": [
-      "<rootDir>/tests/jest/setupTests.js"
+    "name": "imagekitio-react",
+    "version": "1.1.4",
+    "description": "React SDK for ImageKit.io which implements client-side upload and URL generation for use inside a react application.",
+    "main": "dist/cjs/index.js",
+    "module": "dist/esm/index.js",
+    "files": [
+        "dist"
     ],
-    "coveragePathIgnorePatterns": [
-      "<rootDir>/tests/jest/mocks"
-    ],
-    "modulePathIgnorePatterns": [
-      "<rootDir>/tests/cypress/"
-    ]
-  },
-  "author": "ImageKit",
-  "license": "MIT",
-  "devDependencies": {
-    "@babel/cli": "^7.7.4",
-    "@babel/core": "^7.7.4",
-    "@babel/plugin-proposal-class-properties": "^7.7.4",
-    "@babel/plugin-proposal-export-default-from": "^7.7.4",
-    "@babel/preset-env": "^7.7.4",
-    "@babel/preset-react": "^7.7.4",
-    "@babel/register": "^7.7.4",
-    "@babel/runtime": "^7.7.6",
-    "@rollup/plugin-babel": "^5.2.1",
-    "@rollup/plugin-commonjs": "^15.1.0",
-    "@rollup/plugin-json": "^4.1.0",
-    "@rollup/plugin-replace": "^2.3.3",
-    "babel-loader": "^8.0.6",
-    "codecov": "^3.8.0",
-    "cypress": "^5.6.0",
-    "enzyme": "^3.11.0",
-    "enzyme-adapter-react-16": "^1.15.5",
-    "jest": "^26.5.2",
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1",
-    "react-test-renderer": "^16.13.1",
-    "require-context.macro": "^1.2.2",
-    "rollup": "^2.28.2",
-    "rollup-plugin-node-resolve": "^5.2.0",
-    "rollup-plugin-peer-deps-external": "^2.2.3",
-    "rollup-plugin-terser": "^7.0.2",
-    "sinon": "^9.2.0"
-  },
-  "dependencies": {
-    "imagekit-javascript": "^1.3.6",
-    "prop-types": "^15.7.2"
-  },
-  "peerDependencies": {
-    "react": "^16.13.1 || ^17.0.0 || ^18.0.0",
-    "react-dom": "^16.13.1 || ^17.0.0 || ^18.0.0"
-  }
+    "scripts": {
+        "build": "rm -rf dist/ && npm run build:esm && npm run build:cjs",
+        "build:esm": "tsc",
+        "build:cjs": "tsc --module CommonJS --outDir dist/cjs",
+        "cypress:open": "cypress open"
+    },
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/imagekit-developer/imagekit-react"
+    },
+    "author": "ImageKit",
+    "license": "MIT",
+    "devDependencies": {
+        "@types/node": "^17.0.21",
+        "@types/react": "^17.0.37",
+        "@types/react-dom": "^17.0.11",
+        "cypress": "^10.4.0",
+        "react": "^17.0.2",
+        "react-dom": "^17.0.2",
+        "typescript": "^4.5.2"
+    },
+    "peerDependencies": {
+        "react": "^17.0.2",
+        "react-dom": "^17.0.2"
+    },
+    "dependencies": {
+        "cypress-file-upload": "^5.0.8",
```

Comment:
`cypress-file-upload` should be dev dependency.

Reply:
Fixed.